projects/aca-content/src/lib/components/sidenav/sidenav.component.scss

```diff
@@ -140,5 +140,9 @@
         border-bottom: 0;
       }
     }
+
+    .app-expand-menu .aca-expansion-panel {
```

Comment:
we have aca-expansion-panel already in 46th line, can it be reused and styles can be applied there?